PyViCare/PyViCare.py

```diff
@@ -52,21 +52,12 @@ def __extract_devices(self):
         for installation in self.installations:
             for gateway in installation.gateways:
                 for device in gateway.devices:
-                    if device.deviceType not in ["heating", "zigbee", "vitoconnect", "electricityStorage", "EEBus", "hems", "tcu", "ventilation"]:
+                    if device.deviceType not in ["heating", "zigbee", "vitoconnect", "electricityStorage", "tcu", "ventilation"]:
                         continue  # we are only interested in heating, photovoltaic, electricityStorage, hems and ventilation devices
 
                     if device.id == "gateway" and device.deviceType == "vitoconnect":
```

Comment:
Probably this is also no longer necessary.

Reply:
Yes you are right, this is not necessary anymore.